docs/reference/changelog.rst

```diff
@@ -4,6 +4,13 @@ Changelog
 See the `Releases page`_ on GitHub for a complete list of commits that are
 included in each version.
 
```

Comment:
Haven't reviewed Craft Providers in a while, but do we want to start adding refs here so we can point to changelog entries?

Reply:
We probably do, but it's out of scope here. Created this: https://github.com/canonical/craft-providers/issues/775